web/src/components/ThemeContext.tsx

```diff
@@ -30,12 +31,18 @@ type ThemeContainerProps = {
 }
 
 export const ThemeContainer = ({ children, contentDirection }: ThemeContainerProps): ReactElement => {
-  const [themeType, setThemeType] = useState<ThemeType>(() => getSystemTheme())
+  const { value: themeType, updateLocalStorageItem: setThemeType } = useLocalStorage<ThemeType>({
+    key: 'themeType',
```

Comment:
🙃 I think `theme` would be enough here. However, `themeType` is fine as well.
```suggestion
    key: 'theme',
```